src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/ILCompiler.Reflection.ReadyToRun.csproj

```diff
@@ -7,8 +7,8 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <AssemblyKey>Open</AssemblyKey>
     <IsDotNetFrameworkProductAssembly>true</IsDotNetFrameworkProductAssembly>
-    <!-- ILSpy requires this assembly to target netstandard2.0 -->
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <!-- ILSpy requires this assembly to target net7.0 -->
```

Comment:
I don’t think this is going to work. The comment explicitly says that this needs to be netstandard2.0. Why do you need this change? It should be not be needed.

Reply:
> I don’t think this is going to work. The comment explicitly says that this needs to be netstandard2.0. Why do you need this change? It should be not be needed.

As this PR's descrition and the previously replied, there are some building errors if using the `netstandard2.0`

Reply:
> I don’t think this is going to work. The comment explicitly says that this needs to be netstandard2.0. Why do you need this change? It should be not be needed.

As this PR's descrition and the previously replied, there are some building errors if using the `netstandard2.0`

Reply:
@cshung How would you like to see this fixed?